acquisitions/src/main/resources/thunderjet/mod-invoice/eureka/reusable/approve-invoice.feature

```diff
@@ -0,0 +1,18 @@
+Feature: Approve invoice
```

Comment:
By the way, I compared the pay-invoice.feature files and there are also some changes in the params there too. Please take a look at that too.